### PR TITLE
Fix spelling error #57

### PR DESCRIPTION
**Pull request by Vladimir4932**

### CODE REVIEW

**Comment by Vladimir4932** [comment]
@breslavsky check please

**Comment by Vladimir4932** [comment]
@breslavsky check please

File: tutorials/build_conduit.md [hunk 1]
```diff
@@ -573,4 +573,4 @@ docker system prune
 
 # 🙏 Фидбек пожалуйста
 
-<import from="/partials/tutorial_feedback.md"></import>
\ No newline at end of file
+<import from="/partials/tutorial_feedback.md"></import>
```
Comment (breslavsky) [inline tutorials/build_conduit.md:576]: Do we need this changes?

**Comment by Vladimir4932** [comment]
@breslavsky check please

**Pull request merged**